### ISSUE TITLE
Is there a way to restrict poll creation for normal users/participants

### ISSUE THREAD

**Issue by adarshsingh6162** [sampaiodiego/rocket.chat.app-poll#60]
Can we control permission so that only admin can create poll.

**Comment by sampaiodiego** [comment]
No, unfortunately not. But I can see use cases for it, so would be something good to have.. thanks for the suggestion :+1:

**Comment by SouthernGuy454** [comment]
This is something that 100% needs to be added. There are many use cases why an Admin might need this. 
Such as to gather thoughts from users on addition for new channels in a public chat.
